Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -704,7 +703,7 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Pull deltas from Collection SG", "
         encUpdate.endArray();
         encUpdate.endDict();
         for (size_t i = 0; i < collectionCount; ++i) {
-            REQUIRE(_sg.insertBulkDocs(collectionSpecs[i], encUpdate.finish()));
+            REQUIRE(_sg.insertBulkDocs(collectionSpecs[i], encUpdate.finish(), 30.0));
```

Comment:
This 30.0 is not needed explicitly here anymore right?  No need to change just for this PR though.

Reply:
I simply did it so it is clear to the reader of the test what timeout value is being used, understand it’s not really necessary though